platform-configs/include/protectli-vp32xx.robot

```diff
@@ -0,0 +1,14 @@
+*** Settings ***
+Resource    protectli-common.robot
```

Comment:
@JanPrusinowski FYI, I have already tested it before, and the order of these includes matters if we want to allow overriding variables. The order should be reversed actually, as in the `protectli-vp66xx.robot`  for instance:

```
*** Settings ***
Resource    protectli-pro.robot
Resource    protectli-common.robot
```

Reply:
If I am right, in current order, the variable defined in `protectli-common`  would be used, not the one from `protectli-pro`. Can you please verify that as well? Maybe also add explanation to https://github.com/Dasharo/open-source-firmware-validation/blob/develop/docs/adding-new-platforms.md if this ordering really matters?

Reply:
You are right. I have also checked it. When Robot Frameworks looks for for example a key then it searches from top to bottom looking into resources in that order until it finds a key - then it skips searching the rest.
Reversed the order: https://github.com/Dasharo/open-source-firmware-validation/pull/544/commits/fb59cb4b22c73dbb4bc27d50a0a1503776e286f5